docs/static/releasenotes.asciidoc

```diff
@@ -19,6 +20,23 @@ This section summarizes the changes in the following releases:
 * <<logstash-6-1-1,Logstash 6.1.1>>
 * <<logstash-6-1-0,Logstash 6.1.0>>
 
+[[logstash-6-4-3]]
+=== Logstash 6.4.3 Release Notes
+
+* No changes to Logstash-core for 6.4.3
```

Comment:
"Logstash core"

Reply:
@karenzone Does this comment mean you would prefer to have just the hyphen removed, or to have the hyphen removed and "Logstash core" placed in quotes?

Reply:
Just remove hyphen. Thanks for clarifying . I'll be more clear next time